src/preset-themes/jsonSchema/jsonSchema.js

```diff
@@ -0,0 +1,227 @@
+import { makeLog } from "../../lib/utils";
+
+const log = makeLog("background");
```

Comment:
@lmorchard Do you think we need `makeLog(...)`, or since this is more of a CLI script, should we just use `console.log()` to dump to stdout and `process.exitCode`?